apps/ui/src/components/App/Topnav.vue

```diff
@@ -149,10 +149,74 @@ onUnmounted(() => {
         </template>
       </UiButton>
       <IndicatorPendingTransactions />
-      <UiButton class="!px-0 w-[46px]" @click="toggleSkin">
-        <IH-light-bulb v-if="currentMode === 'dark'" class="inline-block" />
-        <IH-moon v-else class="inline-block" />
-      </UiButton>
+      <UiDropdown :z-index="999" :portal="false">
+        <template #button>
+          <slot name="button">
+            <UiButton class="!px-0 w-[46px]" v-bind="$attrs">
+              <IH-cog-6-tooth class="inline-block" />
+            </UiButton>
+          </slot>
+        </template>
+        <template #items>
+          <UiDropdownItem v-slot="{ active }">
+            <a
+              :class="['flex gap-2 items-center', { 'opacity-80': active }]"
+              href="https://snapshot.box"
+            >
+              <IH-bolt :width="16" />
+              Mainnet
+            </a>
+          </UiDropdownItem>
+          <UiDropdownItem v-slot="{ active }">
+            <a
+              :class="['flex gap-2 items-center', { 'opacity-80': active }]"
+              href="https://testnet.snapshot.box"
+            >
+              <IH-beaker :width="16" />
+              Testnet
+            </a>
+          </UiDropdownItem>
+          <UiDropdownItem v-slot="{ active }">
+            <a
+              :class="['flex gap-2 items-center', { 'opacity-80': active }]"
+              href="https://v1.snapshot.box"
+            >
+              <IH-bolt-slash :width="16" />
+              Old interface
+            </a>
```

Comment:
I think It should be similar to what we do on v1
![image](https://github.com/user-attachments/assets/8251a36e-e1fc-4f09-a8cc-02f9e1e12f7f)

It should redirect to current page instead of home page

Reply:
https://github.com/snapshot-labs/snapshot/pull/4910